docs/interfaces/autoupdate-service-examples.txt

```diff
@@ -291,52 +303,86 @@ ModelData: {
     }
 }
 
-IV: structured fields without references
+IV: template fields
 Note:
-    - `B_` is handled as a prefix
+    - `B_$_ids` is a template field, which is not resolved
 
 ModelRequest: {
     collection: "D",
     ids: [1, 2],
     fields: {
         d: null,
-        B_: null,
+        B_$_ids: null
     }
 }
 ModelData: {
     D: {
         1: {
             d: "d1",
+            B_$_ids: ["1", "2", "3"],
+        },
+        2: {
+            d: "d2",
+            B_$_ids: ["1", "4"],
+        }
+    }
+}
+
+V: structured fields without references
```

Comment:
In example IV you renamed it to "template fields". Here you use the term "structured fields" was this intentional?